railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_1.rb.tt

```diff
@@ -29,6 +29,16 @@
 # as equal to an equivalent `Hash` by default.
 # Rails.application.config.action_controller.allow_deprecated_parameters_hash_equality = false
 
+# Active Record Encryption now uses SHA-256 as its hash digest algorithm. Important: If you have
+# data encrypted with previous versions, you should not set the new default or the existing data
+# will fail to decrypt. In this case, if you load the new 7.1 defaults, you need to configure the
+# previous algorithm SHA-1:
+# Rails.application.config.active_record.encryption.hash_digest_class = OpenSSL::Digest::SHA1
+
```

Comment:
```suggestion
#
```

From what I've seen before, if the config is one or the other then there shouldn't be an empty line between them